ffmap/routertools.py

```diff
@@ -372,9 +370,7 @@ def parse_nodewatcher_xml(xml):
 				# skip vpn server
 				if o_out_if == CONFIG["vpn_netif"]:
 					continue
-				elif o_out_if == CONFIG["vpn_netif_l2tp"]:
-					continue
-				elif o_out_if == CONFIG["vpn_netif_l2tp1"]:
+				elif o_out_if.startswith(CONFIG["vpn_netif_l2tp"]):
```

Comment:
This should serve as a wildcard for l2tpX, as requested. Please suggest code for a similar solution to the WAN_uplink.